Formula/libbpg.rb

```diff
@@ -25,6 +25,11 @@ class Libbpg < Formula
   depends_on "libpng"
 
   def install
+    # Work around "-Werror,-Wimplicit-function-declaration" on Xcode 14
+    # The Makefile does not allow modifying CFLAGS with an env variable, so we
+    # have to inject the flag manually
+    inreplace "Makefile", "CFLAGS+=-g", "CFLAGS+=-g -Wno-implicit-function-declaration"
```

Comment:
why not just do `ENV.append_to_cflags "-Wno-implicit-function-declaration"`?

Reply:
See comment?